tests/foreman/ui/test_syncplan.py

```diff
@@ -353,13 +287,15 @@ def test_positive_delete(self, plan_name):
         @Assert: Sync Plan is deleted
 
         """
-        entities.SyncPlan(
-            name=plan_name,
-            interval=SYNC_INTERVAL['day'],
-            organization=self.organization,
-        ).create()
         with Session(self.browser) as session:
-            session.nav.go_to_select_org(self.organization.name)
-            session.nav.go_to_sync_plans()
-            self.syncplan.delete(plan_name)
-            self.assertIsNone(self.syncplan.search(plan_name))
+            for plan_name in generate_strings_list():
+                with self.subTest(plan_name):
+                    entities.SyncPlan(
```

Comment:
OK, bad idea, already discussed this in another PR. Please ignore previous message